nuttx-configs/esc35-v1/bootloader/defconfig

```diff
@@ -296,9 +296,9 @@ CONFIG_ARCH_CHIP_STM32F303CC=y
 # CONFIG_STM32_FLASH_CONFIG_6 is not set
 # CONFIG_STM32_FLASH_CONFIG_8 is not set
 # CONFIG_STM32_FLASH_CONFIG_B is not set
-# CONFIG_STM32_FLASH_CONFIG_C is not set
+# CONFIG_STM32_FLASH_CONFIG_C=y
```

Comment:
@Inspirati 

This looks incorrect to me. I submitted the whole CONFIG_STM32_FLASH_CONFIG stuff upstream to Greg a long while back to handle F4 series parts that have non regular flash page sizes. The F3 has homogeneous 2Kib flahs pages and I believe this does not work as expected 

SIZE=N*pagesize on an F3 works
SIZE=N*pagesize on an F4 is not defined.